README.md

```diff
@@ -26,13 +26,16 @@ const Equity = require('equityjs');
 
 const equity = new Equity({
   consumerKey: 'YOUR_CONSUMER_KEY',
-  consumerSecret: 'YOUR_CONSUMER_SECRET'
+  consumerSecret: 'YOUR_CONSUMER_SECRET',
+  username: 'user', //(optional)
+  password: 'pass', //(optional)
+  grant_type: 'password' //(optional)
```

Comment:
Cool I'll see if I can figure out how to implement it as such. I hadn't thought of it in that context.